input/fsh/profiles/FinnishSchedulingPractitioner.fsh

```diff
@@ -0,0 +1,48 @@
+Profile: FinnishSchedulingPractitioner
```

Comment:
There's probably nothing scheduling specific here?
We should change to use the profile from base profiles.